website_img_dimension/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+Simple module that shows the width and height (including original width and height) of images on the website
```

Comment:
can you elaborate on the use case?
AFAIU it works only when the editor is loaded: you might want to mention this ;)